src/domain/errors.ts

```diff
@@ -13,6 +13,9 @@ export class DuplicateError extends RepositoryError {}
 
 export class BadInputsForFindError extends RepositoryError {}
 export class CouldNotFindError extends RepositoryError {}
+export class CantConnectToMongoDbError extends RepositoryError {
```

Comment:
shouldn't couple domain to `Mongo` implementation. I also don't like leaving out `'` in names like this. `CanNotConnectToDatabaseError` I guess - but its preference not a big deal.

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1707/commits/bd06ee7f110f2a85d1afdca9c34c64bda333b761